components/operators/openshift-serverless/operator/overlays/ibmcloud/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../base
+
+patches:
+  - target:
+      kind: Subscription
+      name: serverless-operator
+    path: patch-channel.yaml  
+  - patch: |-
+      - op: replace
+        path: "/metadata/name"
+        value: openshift-serverless-bh62z
```

Comment:
Does this operatorgroup really have to be openshift-serverless-bh62z or are you just capturing the name for the operatorgroup that was created automatically?

I know that you guys mentioned IBMCloud was doing some weird enforcement on operator group names...